packages/contracts/docs/osx/01-how-it-works/02-framework/02-plugin-management/index.md

```diff
@@ -36,13 +36,13 @@ An Aragon OSx Plugin consist of:
   - Release number
   - Build number
 
-A detailed explanation of the [build and release versioning](../../../02-how-to-guides/02-plugin-development/07-publication/02-versioning.md) is found in the How-to sections in our developer portal.
+A detailed explanation of the [build and release versioning](../../../02-how-to-guides/02-plugin-development/07-publication/01-versioning.md) is found in the How-to sections in our developer portal.
```

Comment:
Make sure that all links you altered work by copying by cloning the developer-portal repo and copying the osx/docs files inside the folder where they belong / the github workflow would put them. 
Then build the page locally (`yarn build && yarn serve`) to get potential warnings / errors.

Otherwise, it could be that the automated follow-up PR on the dev-portal repo has errors and cannot be merged.